### DISCUSSION TITLE
Catch an event when control options is ticked.

### DISCUSSION THREAD

**Discussion by codenamefrance** [Leaflet/Leaflet#9509] [Q&A] [answered]
I implemented a leaflet control where I can manage my overlays (tick boxes).
I'd like to manage when a user activates one of those ticks; is there a way to do it?

**Comment by Falke-Design** [comment, marked as answer]
You can listen on the `overlayadd` and `overlayremove` event:

https://leafletjs.com/reference.html#map-overlayadd

```
map.on('overlayadd overlayremove baselayerchange',(e)=>{
  console.log({enabled: map.hasLayer(e.layer)}, e)
})
```
